test/api-test.js

```diff
@@ -120,6 +120,13 @@ describe('RippleAPI', function() {
       _.partial(checkResult, responses.getTransaction.orderCancellation, done));
   });
 
+  it('getTransaction - trust set', function(done) {
+    const hash =
+      '635A0769BD94710A1F6A76CDE65A3BC661B20B798807D1BBBDADCEA26420538D';
+    this.api.getTransaction(hash, {},
+      _.partial(checkResult, responses.getTransaction.trust, done));
+  });
+
   it('getTransactions', function(done) {
     const options = {types: ['payment', 'order'], outgoing: true, limit: 2};
```

Comment:
It's a tiny thing. Just pointing out that in the account info command, the AccountTxnID uses `initiated` to mean the account's own transactions while getTransactions uses `outgoing`.